docs/reference/cluster/get-desired-balance.asciidoc

```diff
@@ -33,6 +33,8 @@ The API returns the following result:
     "reconciliation_time_in_millis": 0
   },
   "cluster_balance_stats" : {
+    "shard_count": 37,
```

Comment:
Will this number be stable over time?

Reply:
Not sure i follow the question. This number is the total shard count in cluster and it is going to change with every index deletion or creation

Reply:
I was under the impression that we run doc tests that check if the outputs match, isn't it the case here?

Reply:
Oh, I see. This one is skipped with // TEST[skip:Can't reliably test desired balance]